File: d489(triangle_field)/Python/triangle_field.py

from sys import stdin

while True:
    line = stdin.readline()
    if not line:
        break
    a, b, c = map(int, line.split())
    delta = (a + b + c) / 2
    price = (delta * (delta - a) * (delta - b) * (delta - c))
    print(int(price))

File: d489(triangle_field)/C++/triangle_field.cpp
#include <iostream>

int main(int argc, char const *argv[]) {
  int a, b, c;
  while (std::cin >> a >> b >> c) {
    double delta = (a + b + c) / 2;
    double price = delta * (delta - a) * (delta - b) * (delta - c);
    std::cout << (int)price << '\n';
  }
  return 0;
}
